app/views/layouts/_admin_dropdown.html.erb

```diff
@@ -1,178 +1,37 @@
-<li class="dropdown right">
+<%# Privilegies to check for each menu category %>
+<% anvandare = { 'Användare' => [User, MailAlias, Group, Permission] } %>
+<% post = { 'Poster' => [Election, Council, Document] } %>
+<% info = { 'Information' => [Notice, Event, News, BlogPost] } %>
+<% spindel = { 'Spindelman' => [Menu, ShortLink, Constant, Category] } %>
+<% pryl = { 'Pryl' => [Rent, Tool] } %>
+<% ovrigt = { 'Övrigt' => [CafeShift, Introduction, WorkPost, Faq, Page, Album, Contact, Meeting, Song] } %>
+<% all_privilegies = [anvandare, post, info, spindel, pryl, ovrigt] %>
```

Comment:
There's no need to save each menu column to a separate variable, and then "merge" them using arrays. We could do

```
<% menus  = { 'Användare' => [User, MailAlias, Group, Permission],
              'Poster' => [Election, Council, Document],
              'Information' => [Notice, Event, News, BlogPost],
              'Spindelman' => [Menu, ShortLink, Constant, Category],
              'Pryl' => [Rent, Tool],
              'Övrigt' => [CafeShift, Introduction, WorkPost, Faq, Page, Album, Contact, Meeting, Song] } %>
```

and get rid of some code in the loop below as well :)